lnwallet/interface_test.go

```diff
@@ -1998,7 +1998,15 @@ func TestLightningWallet(t *testing.T) {
 
 	rpcConfig := miningNode.RPCConfig()
 
-	chainNotifier, err := btcdnotify.New(&rpcConfig)
+	heightHintDBDir, err := ioutil.TempDir("", "heighthint.db")
```

Comment:
nit: just "heighthint" to not confuse with the actual db file.

Reply:
Fixed.